.github/workflows/release.yaml

```diff
@@ -36,13 +37,15 @@ jobs:
           echo "HELM_REPO=$HELM_REPO" >> $GITHUB_ENV
 
       - name: Publish Helm charts
+        env:
+          RELEASE_ACTOR: ${{ github.actor }}
         run: |
           cd source/deploy/helm
           yq -i '.image.repository = "${{ env.GHCR_REPO }}"' grafana-operator/values.yaml
-          helm registry login -u ${{ github.actor }} -p ${{ secrets.GITHUB_TOKEN }} ghcr.io
-          helm package --app-version ${{ github.ref_name }} --version ${{ github.ref_name }} grafana-operator
+          helm registry login -u "${RELEASE_ACTOR}" -p ${{ secrets.GITHUB_TOKEN }} ghcr.io
```

Comment:
For such things, you can already rely on the set of environment variables that GitHub provides (`GITHUB_ACTOR`,  `GITHUB_REF_NAME`, etc). Not saying that you necessarily have to change this and in other workflows, just wanted to mention that.